### PR TITLE
dev-guide-third-party-support: update GUI tool list

### PR DESCRIPTION
**Pull request by shawn0915**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update gui tool list
related: https://github.com/pingcap/docs/pull/16110

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16110
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-third-party-support.md [hunk 1]
```diff
@@ -177,6 +177,10 @@ PingCAP 与开源社区合作，通过三方工具提供以下支持：
...
-| GUI | 最新已测试版本 | 支持等级 | 教程 |
-| - | - | - | - |
-| [DBeaver](https://dbeaver.io/) | 23.0.1 | Full | N/A |
+| GUI                                                              | 最新已测试版本  | 支持等级   | 教程 |
+|------------------------------------------------------------------|----------------|------------|-----|
+| [JetBrains DataGrip](https://www.jetbrains.com/datagrip/)        | 2023.2.1       | Full       | N/A |
+| [DBeaver](https://dbeaver.io/)                                   | 23.0.3         | Full       | N/A |
+| [Visual Studio Code](https://code.visualstudio.com/)             | 1.72.0         | Full       | N/A |
+| [MySQL Workbench](https://dev.mysql.com/downloads/workbench/)    | 8.0.31         | Compatible | N/A |
+| [Navicat](https://www.navicat.com/en/download/navicat-for-mysql) | 16.3.2         | Compatible | N/A |
```
Comment (Icemap) [inline develop/dev-guide-third-party-support.md:186]:
```suggestion
```

**Review by Icemap — APPROVED** [review]
此处我们需隐藏 `Compatible` 支持等级的工具。Rest LGTM

**Comment by qiancai** [comment]
Hi @shawn0915, as the changes in this PR have been covered by https://github.com/pingcap/docs-cn/pull/16074, would you please close this PR if no further changes? Thanks.

**Comment by shawn0915** [comment]
> Hi @shawn0915, as the changes in this PR have been covered by #16074, would you please close this PR if no further changes? Thanks.

@qiancai @Icemap thank u for your time, close this PR now.

**Pull request closed without merging**